src/ontology/fbbt-edit.obo

```diff
@@ -122569,6 +122585,21 @@ relationship: part_of FBbt:00110893 ! adult midgut region R2
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-06-07T11:33:30Z
 
+[Term]
+id: FBbt:00058222
+name: sugar-sensing neuron of the adult protocerebrum
+def: "Chemosensory neuron in the posterior superior lateral protocerebrum that senses sugars in the hemolymph to regulate feeding. There are 4 of these cells per brain hemisphere. They express the fructose receptor Gr43a and the sugar receptor Gr64a, and secrete corazonin (Miyamato and Amrein, 2014; Fuji et al., 2015)." [FlyBase:FBrf0224445, FlyBase:FBrf0227698]
```

Comment:
FBrf0219961 has more description of these cells (cited in FBrf0224445) and seems to be the original paper describing them, so maybe add this ref too.